src/frontend/components/Felleskomponenter/UtbetalingerModal/UtbetalingerModal.tsx

```diff
@@ -127,7 +124,7 @@ export const UtbetalingerModal: React.FC<UtbetalingerModalProps> = ({
                         label={<TekstBlock block={teksterForPersonType.startdato.sporsmal} />}
                         avgrensMaxDato={periodenErAvsluttet ? gårsdagensDato() : dagensDato()}
                     />
-                    <>
+                    <div>
```

Comment:
Trenger man en `<div> ... </div>` her? `Datovelger` over har det ikke rundt seg så vidt jeg ser

Reply:
`div` brukes i dette tilfellet for å unngå spacing mellom input-feltet og checkbox-en som kommer av `VStack` fra `SkjemaModal`. Med kun `<></>` vil de skyves fra hverandre.

Grunnen til at den første `Datovelger` ikke har `div` rundt seg er fordi den ikke har noen tilhørende checkbox under seg.